# Camplab Wiki
## Meeting schedule
All following meetings take place at the big conference room (E605).

| Recurring meetings | Time | Description |
| --- | --- | --- |
| Camplab lab meeting | Every Tuesday 2:00 pm - 3:00 pm | Lab meeting |
| CBM seminar | Usually every other Tuesday 3:00 pm - 4:30 pm | Each time two students/post-docs/staffs will talk about their research. Check your email for exact date and time. |
| Computational meeting (Spira-Lenburg Lab) | Every Wednesday 1:00 pm - 3:00 pm | Useful for getting an idea about what other students are doing and practising your presentation skills. Talk to Marc if you want to go to this meeting. |
| Single-cell working group meeting | Once a month | Campus-wide seminar. Check email for exact date and time. |

## Slack workspaces
- Camplab: https://camp-lab.slack.com
- Compbiomed: https://compbiomed.slack.com/
- Single Cell Core:

## SCC4 directories
- `/restricted/projectnb/camplab`. Camplab root directory.
- `/restricted/projectnb/camplab/home`. Lab members' own workspaces. Create a folder here and put your stuff in it.
- `/restricted/projectnb/camplab/projects/celda/Datasets/10X`. Publicly available 10X datasets.

## Package development tips
- [R Packages (Hadley Wickham)](http://r-pkgs.had.co.nz/)
- [Coding Style Guide](https://github.com/campbio/celda/wiki/Celda-Development-Coding-Style-Guide)
- [Robust and Efficient Code](https://github.com/campbio/celda/wiki/Celda-Development-Robust-and-Efficient-Code)
- [Rstudio configuration](https://github.com/campbio/celda/wiki/Celda-Development-Rstudio-configuration)
- [FAQ on package development](https://github.com/campbio/celda/wiki/FAQ-on-package-development)
- Vim
  - [Interactive Vim tutorial](https://openvim.com/)
  - [Vim cheetsheet (by Allison McKnight)](https://www.cs.cmu.edu/~15131/f17/topics/vim/vim-cheatsheet.pdf)
- Git
  - [Git cheetsheet](https://about.gitlab.com/images/press/git-cheat-sheet.pdf)
  - [Git mergetool tutorial](https://gist.github.com/karenyyng/f19ff75c60f18b4b8149)
- [Object documentation](http://r-pkgs.had.co.nz/man.html)

## Useful links:
### Single Cell Data Analysis Tutoriols:
- Analysis of single cell RNA-seq data: https://scrnaseq-course.cog.sanger.ac.uk/website/index.html
- Orchestrating Single-Cell Analysis with Bioconductor: https://osca.bioconductor.org/
### Reviews:
#### Single Cell:
- [Papalexi E, Satija R. Single-cell RNA sequencing to explore immune cell heterogeneity. Nat Rev Immunol. 2018;18(1):35-45.](https://www.nature.com/articles/nri.2017.76)
- [Potter SS. Single-cell RNA sequencing for the study of development, physiology and disease. Nat Rev Nephrol. 2018;14(8):479-92.](https://www.nature.com/articles/s41581-018-0021-7)
- [Svensson V, Vento-Tormo R, Teichmann SA. Exponential scaling of single-cell RNA-seq in the past decade. Nat Protoc. 2018;13(4):599-604.](https://www.nature.com/articles/nprot.2017.149)
- [Tanay A, Regev A. Scaling single-cell genomics from phenomenology to mechanism. Nature. 2017;541(7637):331-8.](https://www.nature.com/articles/nature21350)

### Research Computing Services Tutorials
The Research Computing Services (RCS) group, part of Boston University Information Services & Technology, offers a series of tutorials on **programming**, **high performance computing**, and **scientific visualization**. These tutorials are free and open to all members of the Boston University community.
- [Tutorials Schedule](https://www.bu.edu/tech/about/training/classroom/rcs-tutorials/)
- [Tutorial Slide Sets](http://www.bu.edu/tech/support/research/training-consulting/live-tutorials/)

### Marker gene list
Google sheet containing information about marker genes and their corresponding cell types:
https://docs.google.com/spreadsheets/d/1XZ1oAI74ZhCj7gEGgvZUB_ybHF9iFkJGXam6dPTh1gs

## Set up network printer
Instructions: https://www.bumc.bu.edu/it/support/printers/install/

The BU tag number for the printer (Ricoh Aficio MP C4502A) is: **920622**.

## Get an account on the SCC
Ask Josh to add you to the `camplab` project.

More information: http://www.bu.edu/tech/support/research/system-usage/scc-quickstart/#ACCOUNT
